cogs/README.md

```diff
@@ -0,0 +1,69 @@
+# Cogs
+
+Cogs are a way of organizing commands, listeners, and some state into one class. For more information about Cogs, read [the docs](https://discordpy.readthedocs.io/en/stable/ext/commands/cogs.html).
+
+## Alerts
+
+The alerts cog implements the `alerts-add`, `alerts-remove`, and `alerts-list` slash commands. Create alerts for keywords, such as when your name is mentioned.
+
+All alerts are stored in the database created by [`/util/db_builder.py`](/util/db_builder.py).
+
+## DeepL
+
+The DeepL cog implements the `translate` slash command.
+
+If you're confused as to why there are two different `translate` functions, one is to actually translate the text through the DeepL module while the other one is used to run service in a different thread. Note the use of `async`.
+
+## DetectAI (In Progress)
+
+The DetectAI cog implements the `detect-ai` slash command.
+
```

Comment:
Solution

Reply:
could you specify what you mean by solution?

Reply:
oh, i see the mistake. thank you